app/modules/inventory/components/item_row.svelte

```diff
@@ -70,6 +90,34 @@
   </div>
 
   <div class="modes">
+    {#if isNonEmptyArray(itemShelves)}
+      <ul class="shelves-dots">
+        {#each itemShelves as shelf}
+          <li>
+            {#if screen_.isSmall()}
+              <div
+                class="shelf-dot"
+                style="background-image: url({imgSrc(getShelfColor(shelf), 8)})"
+                title={i18n(shelf.name)}
+              >
+              </div>
+            {:else}
+              <a
+                href="/shelves/{shelf._id}"
+                on:click={loadInternalLink}
```

Comment:
your suggestion is still triggering a full reload somehow. You may take a look and commit a fixup directly, otherwise, i would be for merging the PR like and open an issue. 

We could either delete the link for the moment or take this link as a secondary feature (the primary one being to display shelves dots)

Reply:
should be fixed by 72eb18eca